NAMESPACE

```diff
@@ -221,6 +221,7 @@ exportMethods(update)
 import(checkmate)
 importFrom(GenSA,GenSA)
 importFrom(MASS,ginv)
+importFrom(checkmate,test_flag)
```

Comment:
not necessary - as we import the whole package (see line 221)

Reply:
I have just spotted it, thanks!